inc/widgets/odm-contents-same-category-widget.php

```diff
@@ -105,7 +113,7 @@ public function form( $instance ) {
 			<?php endforeach; ?>
 		</p>
 
-		<?php $limit = !empty($instance['limit']) ? $instance['limit'] : -1 ?>
+		<?php $limit = !empty($instance['limit']) ? $instance['limit'] : 5 //-1 ?>
```

Comment:
Ok. But why don't you limited only 5 or 10 default.  It will load too long if someones does't set the number because.